src/lib/utils/column_pruning_utils.cpp

```diff
@@ -26,6 +26,26 @@ std::vector<std::optional<ColumnID>> column_ids_after_pruning(const size_t origi
   return column_id_mapping;
 }
 
+std::unordered_map<ChunkID, ChunkID> chunk_ids_after_pruning(const size_t original_table_chunk_count,
```

Comment:
It is about the expected footprint and access times.
Also, I thought a mapping would be more natural since we want to map from one set to another.
What advantages would a vector have?

Reply:
A vector is small (in case we have many remaining chunks) and access is faster than a hash map (hash map needs collision resolution etc.)

But I guess the footprint is the best argument here.